btc-lsp/src/BtcLsp/Grpc/Client/HighLevel.hs

```diff
@@ -21,30 +21,32 @@ import Proto.BtcLsp (Service)
 import qualified Proto.BtcLsp.Data.HighLevel as Proto
 import qualified Proto.BtcLsp.Data.HighLevel_Fields as Proto
 import qualified Proto.BtcLsp.Method.SwapIntoLn as SwapIntoLn
-import Proto.SignableOrphan ()
 
 swapIntoLn ::
   ( Env m
   ) =>
+  GSEnv ->
   GCEnv ->
   SwapIntoLn.Request ->
   m (Either Failure SwapIntoLn.Response)
-swapIntoLn env req = withRunInIO $ \run ->
+swapIntoLn gsEnv env req = withRunInIO $ \run ->
```

Comment:
Introduce its own signer inside `GCEnv` and use different node for it (alice). Maybe you need to use unlifting to set it.